src/app/components/layout/ChatHistory.js

```diff
@@ -112,7 +112,7 @@ const ChatHistory = ({
 
     // Concatenate with a media URL if there is one
     if (mediaUrl) {
-      preParsedText = `${mediaUrl}\n\n${preParsedText}`;
+      preParsedText = [mediaUrl, preParsedText].filter(contentSection => !!contentSection).join('\n\n');
```

Comment:
This change is to avoid the situation where there is only a `mediaUrl` but no text (so, empty `preParsedText`)... the output was a weird "line with media URL followed by two empty lines".

Reply:
clever